pkg/occlient/occlient.go

```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/fatih/color"
```

Comment:
shouldn't this be in glide.yaml?

Reply:
Yep, that other they might remove it one day. We should have it in glide.yaml if we are explicitly importing it somewhere in our codebase.